cogs/audio.py

```diff
@@ -1786,13 +1787,17 @@ def can_instaskip(self, member):
         """Stops a currently playing song or playlist. CLEARS QUEUE."""
         server = ctx.message.server
         if self.is_playing(server):
-            if self.can_instaskip(ctx.message.author):
-                await self.bot.say('Stopping...')
-                self._stop(server)
+            if ctx.message.author.voice_channel == server.me.voice_channel:
+                if self.can_instaskip(ctx.message.author):
+                    await self.bot.say('Stopping...')
+                    self._stop(server)
+                else:
+                    await self.bot.say("You can't stop music when there are other"
+                                       " people in the channel! Vote to skip"
+                                       " instead.")
             else:
-                await self.bot.say("You can't stop music when there are other"
-                                   " people in the channel! Vote to skip"
-                                   " instead.")
+                await self.bot.reply("you aren't in the current playback"
+                                     " channel.")
```

Comment:
Change this to  
`await self.bot.say("You need to be in the voice channel to stop the music.")`